### DISCUSSION TITLE
BlueRetro App Interface

### DISCUSSION THREAD

**Discussion by ABOhiccups** [darthcloud/BlueRetro#587] [Ideas]
Can you make BlueRetro App Interface for Windows, Mac, Linux, iOS, and Android?

This is for users (like me) who use a different browser that does not support Bluetooth API.

For me. Brave Browser does not support Bluetooth API for privacy reason. Web Browsers for iOS Devices does not support Bluetooth API.

I understand Web Browsers for Android supports Bluetooth API. But, I'm a iPhone user.

I don't use Google Chrome or Microsoft Edge. I personally use Brave Browser on Desktop and Mobile.

BlueRetro App Interface for Non-Web Browser users is a great idea.

**Comment by Sharkeology** [comment]
The best I can suggest is to use an application like Bluefy which does support the Bluetooth API for iPhone.
